.codespell/exclude-file.txt

```diff
@@ -0,0 +1,9 @@
+    of scoped seem allright. I still think there is not enough need
+
+    da de dum, hmm, hmm, dum de dum.
+
+    output=`dmesg | grep hda`
+    p2 = Popen(["grep", "hda"], stdin=p1.stdout, stdout=PIPE)
+
+    Error-de_DE=Wenn ist das Nunstück git und Slotermeyer?
+     Ja! Beiherhund das Oder die Virtualenvironment gersput!
```

Comment:
Is this still needed, or was it just used for testing?

Reply:
It's in PEP 668: https://www.python.org/dev/peps/pep-0668/#marking-an-interpreter-as-using-an-external-package-manager

Reply:
There are multiple ways to silence false positives:
* ignore words
* ignore whole lines (option `exclude-file`)

This is the file containing whole lines to ignore. The benefit of ignoring whole lines is that the ignored false positive might be a true typo in some other line. The downside is that maintenance is more complex.

Reply:
Ah, I see, thanks. I misunderstood the purpose of the `exclude-file` option (and it seems I'm not the only one, @DimitriPapadopoulos themself highlighted multiple instances of user confusion in codespell-project/codespell#2046 )

Unfortunately, it seems I cannot dismiss even my own review comments, so I will delete the other and leave this for you to dismiss.

Reply:
I have also added suggestion https://github.com/codespell-project/codespell/issues/2161 to rename option `exclude-file` to `ignore-lines`.